bucket_api.go

```diff
@@ -93,6 +93,8 @@ func (bucket *Bucket) IsSupported(feature sgbucket.BucketStoreFeature) bool {
 		return true
 	case sgbucket.BucketStoreFeatureN1ql:
 		return false
+	case sgbucket.BucketStoreFeatureMultiXattrSubdocOperations:
+		return true
```

Comment:
Where did we end up with the idea of having an option to test with rosmar with this value set to false?  If there's value in it we could file a ticket for a followup enhancement, but I'm fine just leaving as CBS-only testing if you think that's reasonable.